make_relocatable_python_framework.py

```diff
@@ -61,6 +61,20 @@ def main():
         "Python modules to be installed. If not provided, certain useful "
         "modules for macOS will be installed.",
     )
+    parser.add_option(
+        "--unsign",
+        dest="unsign",
+        action="store_true",
+        help="Unsign binaries and libraries after they are relocatablized, "
+        "in order to avoid having invalid signatures.",
+    )
+    parser.add_option(
+        "--no-unsign",
+        dest="unsign",
+        action="store_false",
+        help="Do not unsign binaries and libraries after they are relocatablized."
+    )
+    parser.set_defaults(unsign=True)
```

Comment:
I don't see the point of having both options. There should be a default behavior, and a single option to flip it to the other. Since the signatures are broken by changing the library references, I'd argue the default behavior should be to remove the signatures on modified files, leaving the `--no-unsign` option available to turn that default behavior off.

So the above changes could be replaced with:
```python
    parser.add_option(
         "--no-unsign",
         dest="unsign",
         action="store_false",
         default=True;
         help="Do not unsign binaries and libraries after they are relocatablized."
     )
```

Reply:
Done, I've removed the --unsign flag in the followup commit.